### DISCUSSION TITLE
How to add a default field in a model and migration?

### DISCUSSION THREAD

**Discussion by rodrigocorreard** [adonisjs/core#4159] [Help]
Is it possible to add a standard field in a model and a migration like createdAt, updatedAT in the construction command of these files? I need to add tenant field in all files generated by default. Thanks

**Comment by WeyIin** [comment]
You would probably need to play with `BaseSchema` and command when migration is generated. I haven't done it myself so I can't really help you much further.

Also adding same fields to all models / migration will create an overhead and data would not be truthful as it has multiple "sources" so to speak. And if one source is updated all others need updating.

**Reply by rodrigocorreard** [reply]
I looked at BaseSchema and ScemaConstrutorContract but I didn't find where to add the fields
